tests/README.md

```diff
@@ -63,6 +63,20 @@ IPA_SERVER_HOST=<ipaserver_host_or_ip> pytest -rs
 
 For a complete list of options check `pytest --help`.
 
+### Types of tests
+
+#### Playbook tests
+
+The playbook tests are the tests the will run our roles / modules with various parameters. Most of those tests will also run same block twice and expect ansible to not report changes ensuring idempotence. In  general those tests don't verify the state of the machine after the playbook is executed.
```

Comment:
Please, change "tests are the tests the will" to "tests will". I'd also change the first phrase with something like "using Ansible" (they do use Ansible, right?).

I think the second phrase should be more direct. Something like "Most of these tests will be executed more than once, to verify module idempotency."